lib/fizzy/execute.cpp

```diff
@@ -79,6 +79,19 @@ std::vector<uint64_t> execute(const module& _module, funcidx _function, std::vec
             stack.push(a + b);
             break;
         }
+        case instr::i32_wrap_i64: {
+            stack.push(static_cast<uint32_t>(stack.pop()));
+            break;
+        }
+        case instr::i64_extend_i32_s: {
+            const auto a = static_cast<int32_t>(stack.pop());
+            stack.push(static_cast<uint64_t>(static_cast<int64_t>(a)));
```

Comment:
```suggestion
            stack.push(static_cast<uint64_t>(int64_t{a}));
```

The `{}` guarantees there is not information loss at conversion.